src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/DirectionsQuestionnaireGenerator.java

```diff
@@ -744,9 +775,14 @@ private Experts getSmallClaimExperts(DQ dq, CaseData caseData) {
         Expert expertDetails;
         if (experts != null) {
             expertDetails = Expert.builder()
-                .name(experts.getExpertName())
+                .name(String.format("%s %s", experts.getFirstName(), experts.getLastName()))
```

Comment:
I think this logic needs to be behind a hnl toggle and if hnl is not enabled you should still be mapping experts.getExpertName().

As it stands if this is merged in and hnl toggle is off in prod it will result in dq being populated with blank names, since in prod it will still be capturing the name under the expertName field until hnl is enabled.

Reply:
i didn't cr the whole pr but this part caught my eye.
i noticed that in your document that you're using for hnl you have firstname lastname instead of just name.
if thats the case, we can just not make any changes to name - so you don't upset the prod cases.

```.name(experts.getExpertName())```